api/v1alpha1/clusterinstance_types.go

```diff
@@ -406,6 +448,27 @@ type ManifestReference struct {
 	Message string `json:"message,omitempty"`
 }
 
+type ReinstallHistory struct {
+	// Observed reinstall generation.
+	// +required
+	ObservedGeneration string `json:"observedGeneration"`
```

Comment:
Should this just be `generation` for the history?

Reply:
yeah good point!